README.md

```diff
@@ -1,14 +1,51 @@
-# Prometheus Integration Pack
+# prometheus integration pack v0.2.2
 
-## Configuration
+> Interact with Prometheus HTTP API
+Daniel Chamot <daniel@nullkarma.com>
+
```

Comment:
Would be nice if this showed details on how to configure the pack

Reply:
Good catch, there was a bug in the template, fixed now.